scene/resources/cylinder_shape.cpp

```diff
@@ -62,6 +62,10 @@ Vector<Vector3> CylinderShape::get_debug_mesh_lines() {
 	return points;
 }
 
+real_t CylinderShape::get_enclosing_radius() const {
+	return Vector2(radius, height).length();
```

Comment:
For the radius of an enclosing sphere, shouldn't it be `Vector2(radius, height*0.5).length()`?

Reply:
Yep, fixed